test/modules/encoders/test_clip_resnet_encoder.py

```diff
@@ -4,29 +4,34 @@
 # This source code is licensed under the BSD-style license found in the
 # LICENSE file in the root directory of this source tree.
 
-import unittest
+import pytest
 
 import torch
+from test.test_utils import assert_expected
 from torchmultimodal.modules.encoders.clip_resnet_encoder import ResNetForCLIP
 from torchmultimodal.utils.common import get_current_device
 
 
-class TestCLIPModule(unittest.TestCase):
-    def setUp(self):
+class TestResnetEncoder:
+    @pytest.fixture(autouse=True)
+    def set_seed(self):
```

Comment:
Although there's just one test class, having the seed fixture at the script level with `autouse` will ensure that each method (regardless of which TestClass it belongs to) will get a fresh seed. 

device could be universal fixture --- would recommend putting it at the script level as well.

Use `test.test_utils.set_rng_seed` here.